api/src/pcapi/settings.py

```diff
@@ -214,6 +214,7 @@
 RECAPTCHA_API_URL = "https://www.google.com/recaptcha/api/siteverify"
 RECAPTCHA_SECRET = secrets_utils.get("RECAPTCHA_SECRET")
 NATIVE_RECAPTCHA_SECRET = secrets_utils.get("NATIVE_RECAPTCHA_SECRET")
+RECAPTCHA_IGNORE_VALIDATION = bool(os.environ.get("RECAPTCHA_IGNORE_VALIDATION", False))
```

Comment:
```python
>>> bool("false")
True
```

Oups, ça n'est probablement pas ce qui est attendu par l'utilisateur. :) Pour tous les settings booléens (celui-ci, `DEBUG_ACTIVATED`, etc.), je serais plutôt d'avis d'utiliser `bool(int(...))`. Ainsi, l'utilisateur doit passer 1 (pour activer) ou 0 (pour désactiver), et toute autre valeur lui explose assez explicitement au visage.

```suggestion
RECAPTCHA_IGNORE_VALIDATION = bool(int(os.environ.get("RECAPTCHA_IGNORE_VALIDATION", 0)))
```